sounds/views.py

```diff
@@ -798,6 +798,28 @@ def embed_iframe(request, sound_id, player_size):
     return render_to_response('sounds/sound_iframe.html', locals(), context_instance=RequestContext(request))
 
 
+def oembed(request):
+    url = request.GET.get('url', None)
+    view, args, kwargs = resolve(urlparse(url)[2])
+    if not 'sound_id' in kwargs:
+        raise Http404
+    sound_id = kwargs['sound_id']
+    sound = get_object_or_404(Sound, id=sound_id, moderation_state='OK', processing_state='OK')
+    player_size = request.GET.get('size', 'medium')
+    if player_size == 'large':
+        sizes = [920, 245]
```

Comment:
we should probably remove these from here and from the template and make them constants somewhere...